modules/tf-cloud-credential/variables.tf

```diff
@@ -0,0 +1,22 @@
+variable "workspace_name_prefix" {
+  type        = string
+  description = "The name prefix to use for the workspace"
+}
+
+variable "organization" {
+  type        = string
+  description = "The workspace organization"
+}
+
+variable "iam_access_key" {
+  type = object({
+    id     = string
+    secret = string
+  })
+  description = "The aws_iam_access_key to use as credentials for workspace."
```

Comment:
Should be: `"The aws_iam_access_key id/secret pair to use as credentials for the workspace."`

Reply:
I also resolved this in the latest commit for this PR